ci/ci/github.py

```diff
@@ -873,19 +876,28 @@ async def _update_deploy(self, batch_client, db: Database):
                 else:
                     self.deploy_state = 'failure'
 
+                if not is_test_deployment and self.deploy_state == 'failure':
```

Comment:
Keeping Both

LN 879:
<<<<<<< HEAD
=======
                if not is_test_deployment and self.deploy_state == 'failure':
                    url = deploy_config.external_url('ci', f'/batches/{self.deploy_batch.id}')
                    deploy_failure_message = f"""
state: {self.deploy_state}
branch: {self.branch.short_str()}
sha: {self.sha}
url: {url}
"""
                    await send_zulip_deploy_failure_message(deploy_failure_message, db, self.sha)
>>>>>>> b7bde56d5

Reply:
Sorry, I should have talked about using `merge.conflictStyle = zdiff3` in your Git config to get the three-part version of these conflicts.

Try the `git diff …… -- ci/ci/github.py` trick for this file and see if you want to change your mind about what happened on these lines :smile: